pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -321,18 +404,34 @@ func (h *HandlerImpl) updateAllocatedDevices(activePods []*v1.Pod) {
 	if len(podsToBeRemoved) <= 0 {
 		return
 	}
-	glog.V(5).Infof("pods to be removed: %v", podsToBeRemoved.List())
+	glog.V(3).Infof("pods to be removed: %v", podsToBeRemoved.List())
 	h.podDevices.delete(podsToBeRemoved.List())
 	// Regenerated allocatedDevices after we update pod allocation information.
 	h.allocatedDevices = h.podDevices.devices()
 }
 
+// checkpointData struct is used to store pod to device allocation information
+// and registered device information in a checkpoint file.
+// TODO: add version control when we need to change checkpoint format.
```

Comment:
Add a version field today please, or it will lead to a lot of troubles in future.